joomla/joomla-cms
# [4.0] Quickicons - Add Shortcut to Template Code

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/25916

### Summary of Changes
Added Template Code Quickicon
Changed Language Keys of Template Style Icon 
Added Language Keys for Template Code

### Testing Instructions
Apply the patch
Activate the Quick Icon in the module settings
Click the Template Code Link to see if you end up where you should
